src/test/java/org/folio/innreach/controller/d2ir/InnReachCirculationControllerTest.java

```diff
@@ -1052,13 +1052,18 @@ private void assertPatronHoldFieldsAreNull(TransactionPatronHold hold) {
     assertNull(hold.getFolioItemBarcode());
   }
 
-  @Test
+  @ParameterizedTest
+  @EnumSource(names = {"PATRON_HOLD", "TRANSFER"})
   @Sql(scripts = {
     "classpath:db/central-server/pre-populate-central-server.sql",
     "classpath:db/inn-reach-transaction/pre-populate-inn-reach-transaction.sql"
   })
-  void checkTransactionIsNotInStateItemInTransitOrReturnUncirculated() {
+  void checkTransactionIsInStatePatronHoldOrTransfer(InnReachTransaction.TransactionState testEnums) {
```

Comment:
what is testEnums? lets rename it to state

Reply:
renamed